src/views/pages/Swap/Swap.js

```diff
@@ -553,6 +548,13 @@ const Swap = () => {
     return '0'
   }
 
+  const getRateSlip = () => {
```

Comment:
Add calc function for USD-value (internally derived) rate-slip between the two assets